cmd/zt_copy_s2smigration_test.go

```diff
@@ -50,7 +50,9 @@ var defaultS2SInvalideMetadataHandleOption = common.DefaultInvalidMetadataHandle
 
 func (s *cmdIntegrationSuite) SetUpSuite(c *chk.C) {
 	s3Client, err := createS3ClientWithMinio(createS3ResOptions{})
-	c.Assert(err, chk.IsNil)
+	if err != nil {
+		return
+	}
```

Comment:
please add a comment to explain why? It's hard to know/remember the reason otherwise

Reply:
Done